src/renderers/WebGLRenderer.js

```diff
@@ -1173,6 +1173,8 @@ function WebGLRenderer( parameters ) {
 
 		projectObject( scene, camera, 0, _this.sortObjects );
 
+		currentRenderList.cleanup();
```

Comment:
How is this invocation possible without exposing `cleanup()`?

Reply:
Bad cherry pick. It seemed so trivial, I was not not careful enough. I hope I have it right this time.